ipvs.go

```diff
@@ -204,3 +222,21 @@ func (i *Handle) GetConfig() (*Config, error) {
 func (i *Handle) SetConfig(c *Config) error {
 	return i.doSetConfigCmd(c)
 }
+
+// GetInfo returns info details from IPVS
+func (i *Handle) GetInfo() (*Info, error) {
+	res, err := i.doGetInfoCmd()
+	if err != nil {
+		return nil, err
+	}
+
+	ver := uint(res.version)
+	return &Info{
+		Version: &Version{
```

Comment:
Can we put the conversion of `ver` to `Version` in a separate function so we can add unit tests?

Reply:
`version` as one attribute of `ipvsInfo`, we should test it as a whole? like test case `TestInfo`

Reply:
@kwanhur I agree with @andrewsykim.
We can test different inputs for the conversion and not rely on the netlink of the test system.